unison-syntax/src/Unison/Syntax/Parser.hs

```diff
@@ -416,23 +416,28 @@ string = queryToken getString
     getString (L.Textual s) = Just (Text.pack s)
     getString _ = Nothing
 
-tupleOrParenthesized :: (Ord v) => P v m a -> (Ann -> a) -> (a -> a -> a) -> P v m a
-tupleOrParenthesized p unit pair = seq' "(" go p
+-- | Parses a tuple of 'a's, or a single parenthesized 'a'
+--
+-- returns the result of combining elements with 'pair', alongside the annotation containing
+-- the full parenthesized expression.
+tupleOrParenthesized :: Ord v => P v m a -> (Ann -> a) -> (a -> a -> a) -> P v m (Ann {- spanAnn -}, a)
```

Comment:
The annotations for these was incorrect and was setting the annotation for EACH member to be the whole span of the brackets. Should be fixed up now.